packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -45,6 +45,12 @@ function drawNative(nodeToData: Map<HostInstance, Data>, agent: Agent) {
 function drawWeb(nodeToData: Map<HostInstance, Data>) {
   if (canvas === null) {
     initialize();
+  } else {
+    try {
+      if (canvas.hasAttribute('popover') && !canvas.matches(':popover-open')) {
+        canvas.showPopover();
+      }
```

Comment:
Are you gating it with try/catch because for feature checking?

We might bump the minimum required version for Chrome / other browsers here:
https://github.com/facebook/react/blob/main/packages/react-devtools-extensions/chrome/manifest.json#L7

So you won't need to worry about if its supported or not

Reply:
I've added Edge as well! Yes, Firefox requires 125 or higher, but it's already set to 128, so we're good.